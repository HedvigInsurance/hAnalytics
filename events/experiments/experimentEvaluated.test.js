const validateEvent = require("../../commons/validateEvent.js")

test('validate variant', () => {
    const event = validateEvent(__filename, {
        name: "chat_provider",
        enabled: true,
        variant: "bot-service"
    })

    expect(event).toMatchSnapshot()
});

test('validate non variant', () => {
    const event = validateEvent(__filename, {
        name: "is_something_active",
        enabled: true,
        variant: null
    })

    expect(event).toMatchSnapshot()
});