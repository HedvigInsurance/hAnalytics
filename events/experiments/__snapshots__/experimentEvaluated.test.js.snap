// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`validate non variant 1`] = `
Object {
  "name": "experiment_evaluated",
  "variables": Object {
    "enabled": true,
    "name": "is_something_active",
    "variant": null,
  },
}
`;

exports[`validate variant 1`] = `
Object {
  "name": "experiment_evaluated",
  "variables": Object {
    "enabled": true,
    "name": "chat_provider",
    "variant": "bot-service",
  },
}
`;
